keyboards/melgeek/z70ultra/config.h

```diff
@@ -18,19 +18,20 @@
 
 #include "config_common.h"
 
-/* USB Device descriptor parameter */
-#define VENDOR_ID       0xEDED
-#define PRODUCT_ID      0x6570
-#define DEVICE_VER      0x0001
-#define MANUFACTURER    MelGeek
-#define PRODUCT         Z70Ultra
+
 
 /* COL2ROW, ROW2COL*/
 #define DIODE_DIRECTION COL2ROW
 
 /* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
 #define DEBOUNCE 3
 
+/* disable these deprecated features by default */
+#define _CHIBIOS_RT_CONF_VER_6_1_
+#define _CHIBIOS_HAL_CONF_VER_7_1_
```

Comment:
I'm not sure that these are okay, here.

Reply:
Some documents were missing in the previous project, so I used them. It doesn't matter whether they are there or not